2/contrib/s2i/run

```diff
@@ -6,6 +6,11 @@
 #
 # It also sets the admin password to ${JENKINS_PASSWORD}.
 #
+
+# This value needs to be updated to reflect the current release version
+# to ensure we properly migrate old slave configurations to the new tag.
+JENKINS_SLAVE_IMAGE_TAG=${JENKINS_SLAVE_IMAGE_TAG:-v3.9}
```

Comment:
can this be wired through directly from the release.version file?  E.g. ADD it via the Dockerfiles, then do `JENKINS_SLAVE_IMAGE_TAG=${JENKINS_SLAVE_IMAGE_TAG:-$(cat path/to/release.version)}`?

Reply:
I was looking at doing this but the short answer is "not easily" and i don't want to make things too confusing for the dist-git repo that's going to have to build this for rhel.

Reply:
If it's not wired through, would it be advisable to have some test to ensure that the two are in sync?  Is that any nicer?

Reply:
for now i've worked around the symlink problem by making the "real" file live in the "2" directory and the root one be the symlink.

If we ever create a "3" directory we'll have to revisit how this is being done.

Reply:
(you can't add a symlink from within a dockerfile if the symlink points outside the docker build context dir)